Attempted default import breaks game

**Bug Summary**
In `src/features/monsters/actions/attack-player.js`, there is an attempted default import from `utils/dice.js` despite there being no default exports from that file.

**Test Case(s)**
N/A

---

**Expected behavior**
The game runs without errors or warnings.

**Observed Behaviour**
The game doesn't run.

**Steps To Reproduce**
<!-- The steps performed to reproduce the bug -->
1. Run the game using `yarn start`
2. Observe

**Environment**
- Version: 96b4a57
- OS: Ubuntu
- Browser: N/A

---

**Code Examples**
N/A

**Stack Trace**
```
yarn run v1.22.4
$ react-scripts build
Creating an optimized production build...
Failed to compile.

./src/features/monsters/actions/attack-player.js
Attempted import error: '../../../utils/dice' does not contain a default export (imported as 'calculateDamage').


error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
##[error]Process completed with exit code 1.
```


**Screenshots**
N/A

**Error Report**
N/A

---

**Additional context**
The failure occurred during deployment.

--- a/src/features/monsters/actions/attack-player.js
+++ b/src/features/monsters/actions/attack-player.js
@@ -1,4 +1,4 @@
-import calculateDamage from '../../../utils/dice';
+import { calculateDamage } from '../../../utils/dice';
 
 export default function attackPlayer(attackValue, dice) {
     return (dispatch, getState) => {

